go/libkb/sig_chain.go

```diff
@@ -490,6 +490,13 @@ func (sc *SigChain) verifySubchain(ctx context.Context, kf KeyFamily, links []*C
 			}
 		}
 
+		if pgprev, ok := tcl.(*RevokeChainLink); ok {
+			for _, revokedKid := range pgprev.GetRevokeKids() {
```

Comment:
More needs to be done here... I didn't know if it's possible to easily go from `[]keybase1.SigID` to kids with `pgprev.GetRevocations()`